src/Uno.UI/UI/Xaml/Controls/MenuFlyout/CommandingHelpers.cs

```diff
@@ -0,0 +1,226 @@
+			 ICommand uiCommand,
+			 FrameworkElement target,
+			 DependencyProperty labelProperty)
+		{
+			string localLabel = null;
+			var localLabelAsI = target.ReadLocalValue(labelProperty);
+
+			if (localLabelAsI != null)
+			{
+				localLabel = localLabelAsI.ToString();
+			}
+
+			Console.WriteLine($"BindToLabelPropertyIfUnset: [{localLabelAsI}]/[{localLabel}]");
+
+			if (localLabelAsI == DependencyProperty.UnsetValue || string.IsNullOrEmpty(localLabel))
+			{
+				target.SetBinding(labelProperty, new Binding { Path = "Label", Source = uiCommand });
+			}
+		}
+
+		internal static void BindToIconPropertyIfUnset(
+			 XamlUICommand uiCommand,
+			 FrameworkElement target,
+			 DependencyProperty iconProperty)
+		{
+			IconElement localIcon;
+			object localIconAsI = target.ReadLocalValue(iconProperty);
+
+			localIcon = localIconAsI as IconElement;
+
+			if (localIconAsI == DependencyProperty.UnsetValue || localIcon == null)
+			{
+				IconSourceToIconSourceElementConverter converter = new IconSourceToIconSourceElementConverter();
+				target.SetBinding(iconProperty, new Binding { Path = "IconSource", Source = uiCommand, Converter = converter });
+			}
+		}
+
+		internal static void BindToIconSourcePropertyIfUnset(
+			 XamlUICommand uiCommand,
+			 FrameworkElement target,
+			 DependencyProperty iconSourceProperty)
+		{
+			object localIconSourceAsI;
+			IconSource localIconSource;
+			localIconSourceAsI = target.ReadLocalValue(iconSourceProperty);
+
+			localIconSource = localIconSourceAsI as IconSource;
+
+			if (localIconSourceAsI == DependencyProperty.UnsetValue || localIconSource == null)
+			{
+				target.SetBinding(iconSourceProperty, new Binding { Path = "IconSource", Source = uiCommand });
+			}
+		}
+
+		internal static void BindToKeyboardAcceleratorsIfUnset(
+			 XamlUICommand uiCommand,
+			 UIElement target)
+		{
+			IList<KeyboardAccelerator> targetKeyboardAccelerators;
+			int targetKeyboardAcceleratorCount;
+
+			targetKeyboardAccelerators = target.KeyboardAccelerators;
+			targetKeyboardAcceleratorCount = targetKeyboardAccelerators.Count;
+
+			if (targetKeyboardAcceleratorCount == 0)
+			{
+				var converter = new KeyboardAcceleratorCopyConverter();
+				target.SetBinding(UIElement.KeyboardAcceleratorsProperty, new Binding { Path = "KeyboardAccelerators", Source = uiCommand, Converter = converter });
+			}
+		}
+
+		internal static void BindToAccessKeyIfUnset(
+			 XamlUICommand uiCommand,
+			 UIElement target)
+		{
+			string localAccessKey;
+			localAccessKey = target.AccessKey;
+
+			if (localAccessKey == null || string.IsNullOrEmpty(localAccessKey))
+			{
+				target.SetBinding(UIElement.AccessKeyProperty, new Binding { Path = "AccessKey", Source = uiCommand });
+			}
+		}
+
+		internal static void BindToDescriptionPropertiesIfUnset(
+			 XamlUICommand uiCommand,
+			 FrameworkElement target)
+		{
+			string descriptionFromCommand;
+			descriptionFromCommand = uiCommand.Description;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'descriptionFromCommand'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)